packages/shared/components/inputs/Input.svelte

```diff
@@ -16,27 +17,25 @@
     export let disabled = false
 
     let inputElement
+    let decimalSeparator = getDecimalSeparator()
 
     const handleInput = (e) => {
         value = e.target.value
     }
 
     const onKeyPress = (e) => {
-        if (e.keyCode !== 8) {
-            const isReturn = e.keyCode === 13
-            if (isReturn && submitHandler) {
+        if (e.key !== 'Tab') {
+            const isEnter = e.key === 'Enter'
+            if (isEnter && submitHandler) {
```

Comment:
Nice, we should make sure we use `key` instead of `keyCode` in other places as well (I can make a PR when I have some time)

Reply:
This is a nice web page which lets you type and it gives you the equivalent keys etc https://keycode.info/
Using `key` for the separator and digits means we also support number pads